internal/service/location/storage/storage.go

```diff
@@ -58,8 +60,8 @@ func (s *Storage) LastLocation(ctx context.Context, areaKey string, workerID str
 	if len(v) > 0 && v[0] != nil {
 		return &LastLocation{
 			WorkerID:     workerID,
-			Longitude:    v[0].Longitude,
-			Latitude:     v[0].Latitude,
+			Longitude:    util.Round6(v[0].Longitude),
+			Latitude:     util.Round6(v[0].Latitude),
```

Comment:
Why not keep all decimals for maximum precision?

Reply:
round6 is just for clarity, it breaks nothing, but easier for human eye. i spotted 6 digits round used by LatLon converters (somewhere in the web), also google maps seem to limit it to even 5 digits.